whisper-fine-tuning-sprint/README.md

```diff
@@ -0,0 +1,635 @@
+# Whisper fine-tuning sprint 🤗
+
+Welcome to the Whisper fine-tuning sprint 🎙️ !
+
+The goal of this event is to build **robust**, **real-world** speech recognition (ASR) systems in as many languages as possible 🌏🌍🌎.
+If necessary and available, free access to a V100s 32 GB GPU will kindly be provided by the [PUT CLOUD PROVIDER HERE](https://PUTCLOUDPROVIDERHERE.COM) 🚀.
+This document summarizes all the relevant information required for the speech community event 📋.
+
+To sign-up, please see [this forum post](https://LINK_ACTUAL_FORUM) 🤗. Please make sure to:
+- Read it in detail
+- Fill the [google form](https://forms.gle/F2bpouvhDpKKisM39)
+- Join the [Hugging Face Discord server](https://hf.co/join/discord) and make sure you have access to the #events channel.
+
+[comment]: # TODO: VB - create a post on the forum and link it above. Update the URL to cloud provider when decided.
+
+## Table of Contents
+
+- [TLDR;](#tldr)
+- [Important dates](#important-dates)
+- [How to install pytorch, transformers, datasets](#how-to-install-relevant-libraries)
+- [Data and Preprocessing](#data-and-preprocessing)
+- [How to fine-tune an acoustic model](#how-to-finetune-an-acoustic-model)
+- [How to fine-tune with OVH could](#how-to-finetune-with-ovh-cloud)
+- [How to combine n-gram language models with acoustic model](#how-to-combine-n-gram-with-acoustic-model)
+- [Evaluation](#evaluation)
+- [Prizes](#prizes)
+- [Communication and Problems](#communication-and-problems)
+- [Talks](#talks)
+- [General Tips & Tricks](#general-tips-and-tricks)
+
+## TLDR
+
+Participants are encouraged to leverage pre-trained speech recognition checkpoints,
+preferably [openai/whisper-large](https://huggingface.co/openai/whisper-large), 
```

Comment:
Maybe copy and past the table from the blog with checkpoints and params? https://huggingface.co/blog/fine-tune-whisper#introduction

Also probably medium is going to be the largest we can feasibly train on an A100 -> this is fine as medium performs very similarly to large on most languages

We can train large but it'll be very slow (high number of gradient accumulation steps) -> I think we're better off encouraging the smaller checkpoints (small and medium) for more training steps